main/src/main/java/cgeo/geocaching/connector/gc/GCParser.java

```diff
@@ -1597,22 +1601,45 @@ private static void getExtraOnlineInfo(@NonNull final Geocache cache, final Stri
         final String userToken = parseUserToken(page);
         final Observable<LogEntry> logs = getLogs(userToken, Logs.ALL);
         final Observable<LogEntry> ownLogs = getLogs(userToken, Logs.OWN).cache();
-        final Observable<LogEntry> specialLogs = Settings.isFriendLogsWanted() ?
-                Observable.merge(getLogs(userToken, Logs.FRIENDS), ownLogs) : Observable.empty();
-        final Single<List<LogEntry>> mergedLogs = Single.zip(logs.toList(), specialLogs.toList(),
-                (logEntries, specialLogEntries) -> {
-                    mergeFriendsLogs(logEntries, specialLogEntries);
-                    return logEntries;
-                }).cache();
-        mergedLogs.subscribe(logEntries -> DataStore.saveLogs(cache.getGeocode(), logEntries, true));
```

Comment:
This is unrelated to this PR, but why the hell is a Parser class manipulating the database???

Reply:
Because it is manipulating e.g. the visited date of the cache...

Maybe we need to rework the workflow, so updating the visited date is done outside

Reply:
Exactly, update should happen outside of the parser.
But as said, this is unrelated to your PR

Reply:
But makes it probably easier for me

Reply:
Maybe I can move the method from GCParser to GCConnector, but this doesn't make it much better...